wcivf/apps/elections/templates/elections/includes/_how-to-vote.html

```diff
@@ -12,7 +12,7 @@
 <details style="padding-top:1em; clear: both;">
   <summary>How you vote</summary>
   {% if voting_system.slug == "FPTP" or "sv" or "AMS" %}
-    This election uses <a href="{{ voting_system.get_absolute_url }}">{{ voting_system.name }}</a>.
+    This election uses <a href="{{ voting_system.get_voting_system_url }}">{{ voting_system.name }}</a>.
```

Comment:
Was this change in relation to https://github.com/DemocracyClub/WhoCanIVoteFor/issues/749? I can't see a `get_voting_system_url` method - so although you won't get a 500 error anymore, that will be because `get_voting_system_url` will silently fail, but the link to the explainer pages will no longer work either.

The cause of the error is the `if` statement on line 14. At the moment the `if` statement will always resolve to True, so we always try to return a URL for an explainer, but for some voting systems where we don't have a page to link to (like STV) the `ValueError` ends up being raised. When you use `or` you need to use the first part of the comparison again each time so it becomes:
```
{% if voting_system.slug == "FPTP" or voting_system.slug == "sv" or voting_system.slug == "AMS" %}
```

But I would also change the `get_absolute_url` method on VotingSystem to return `None` or an empty string instead of raising a ValueError:
```
    @property
    def get_absolute_url(self):
        if self.slug == "FPTP":
            return reverse("fptp_voting_system_view")
        elif self.slug == "AMS":
            return reverse("ams_voting_system_view")
        elif self.slug == "sv":
            return reverse("sv_voting_system_view")
        return None
```
And then the if statement in the template can be changed to instead check if we have a URL to link to or not:
```
{% if voting_system.get_absolute_url %}
```

Reply:
Thanks, all that makes sense.

Reply:
I'm a little confused by this!

I think we can keep the name `get_absolute_url` if we're returning `None` now. Also, if we're returning `None then we can simplify the conditional in the template, and simply do `{% if voting_system.get_absolute_url %}` - that way when we add a new explainer we don't need to change the template in order to link to it.

Reply:
We already have that name in the election model.

Reply:
Also, sorry about suggesting the `ValueError` in the previous version - I had assumed we had an explainer for every known voting system, so an exception would be useful to us!